### ISSUE TITLE
Upgrading to loolwsd 6.4.13-1 leads to JavaScript error (Uncaught ReferenceError: Cursor is not defined)

### ISSUE THREAD

**Issue by steffen-moser** [CollaboraOnline/online#3369]
**Describe the bug**
We are running Nextcloud 20.0.12 combined with the Collabora Online App 3.8.6. After upgrading our CODE server (Ubuntu 20.04) from "collaboraoffica 6.4.10.48-48" to "collaboraoffice 6.4.10.49-49", CODE seems to be broken. 

Whenever we try to open a document, we run into these error messages on the browser console:

```
bundle.js:1 Uncaught ReferenceError: Cursor is not defined
    at NewClass.initialize (bundle.js:1)
    at new NewClass (bundle.js:1)
    at Object.L.map (bundle.js:1)
    at bundle.js:1
    at bundle.js:1
    at bundle.js:1
```

Other people do seem to encounter exactly the same problem, cf.: [1]

It does not seem to be related to Nextcloud. The same error is displayed when accessing Collabora Online from Moodle. Of course, we empty our browser cache, I even used a plain new profile. We do not find any hints in the logs, neither in Nextcloud's logs nor in Collabora's logs.

**To Reproduce**
1. Install the most recent version of CODE on Ubuntu 20.04.
2. Try to access CODE from Nextcloud or Moodle by opening an existing document (or creating a new document). 

**Expected behavior**
The document should be opened.

**Actual behavior**
An empty screen is shown, as depicted in [1]. The browser console shows the above-mentioned JavaScript error.

**Desktop (please complete the following information)**
 - OS: macOS 11.6
 - Browser: Chrome
 - Version: 94.0.4606.81
 
Other browsers (Chrome on Android, Firefox on macOS, Safari on macOS, Edge on macOS) show the same problem.

**Additional context**
We did not have any problem with "collaboraoffice 6.4.10.48-48". We did not change anything on the Nextcloud or Moodle side.

[1] https://help.nextcloud.com/t/cant-use-collabora-on-nextcloud/124345

**Comment by steffen-moser** [comment]
Could it be possible that "/usr/share/loolwsd/loleaflet/dist/bundle.js" is just incomplete in 6.4.13-1? 

It seems that the function "Cursor" is missing there (I've done a client-side comparison of the "bundle.js" files the browser receives from a (working) 6.4.11 and a (non-working) 6.4.13 server).

**Comment by jlebleu** [comment]
Hello same problem after upgrading to 6.4.13 unfortunatly I don't know how to go back and collabora is not working any more, any chance to get the 6.4.11 package somewhere ?

**Comment by bernd-wechner** [comment]
And same problem here. A fix would be most appreciated. Collabara gone down since upgrade. :-(

**Comment by canuckdev** [comment]
Same problem here:
`
20.04.3 LTS (Focal Fossa) x86_64
collaboraoffice6.4 6.4.10.49-49
loolwsd 6.4.13-1
`

**Comment by thebearon** [comment]
For some reason `bundle.js` in the Ubuntu 20.04 `loolwsd` package was incomplete. The package has been updated, and is now working. Sorry for the inconvenience!

**Comment by bernd-wechner** [comment]
> @bernd-wechner In case your question is still relevant, I was using a Docker image and just pulled an older tag.

Thanks. Wouldn't have helped me as I had the loolwsd package installed on Ubuntu not a docker image. But it's all come good now with 6.4.13-2.

**Comment by fxandrei** [comment]
This problem still exists it seems.
Ive been trying for 2 weeks to find a solution, and nothing.
Reinstalled multiple version in multiple configs (full vm, container, etc), still no luck.
The bundle.js file is incomplete.

Is there any way to force this file to be sent to the client (browser), or maybe host it somewhere else ?

**Comment by steffen-moser** [comment]
Which version of loolwsd/coolwsd do you want to use?

**Comment by fxandrei** [comment]
> Which version of loolwsd/coolwsd do you want to use?

Well i dont want to use any particular version.
I just updated to the last version.
Ive been having this issue for 2-3 weeks.

The strange thing is that it worked in the beginning, and im not sure why.

I have also made a forum post here:
https://forum.collaboraonline.com/t/i-have-no-idea-why-this-is-happening-and-what-to-check-not-loading-on-some-clients/1115/6

Anyway, the version im using now (since the update yesterday) is 21.06.30-1 .
